docker/ansible/ansible-container.yaml

```diff
@@ -103,8 +104,8 @@ node_templates:
           command: { get_attribute: [docker_ansible_container_files, ansible_container_command_arg] }
           working_dir: { get_input: container_volume }
           environment:
-            ANSIBLE_HOST_KEY_CHECKING: False
-            ANSIBLE_INVALID_TASK_ATTRIBUTE_FAILED: False
+            ANSIBLE_HOST_KEY_CHECKING: false
+            ANSIBLE_INVALID_TASK_ATTRIBUTE_FAILED: false
```

Comment:
I believe that because these are environment variables that they must be "False". Did you test that this blueprint works still?